--- lxc-1.0.5/config/init/systemd/Makefile.am.0003~	2014-08-29 10:25:23.586235868 +0200
+++ lxc-1.0.5/config/init/systemd/Makefile.am	2014-08-29 10:26:47.246189975 +0200
@@ -3,7 +3,7 @@ EXTRA_DIST = \
 	lxc.service
 
 if INIT_SCRIPT_SYSTEMD
-SYSTEMD_UNIT_DIR = $(prefix)/lib/systemd/system
+SYSTEMD_UNIT_DIR = /lib/systemd/system
 
 lxc-autostart-helper: ../sysvinit/lxc.in $(top_builddir)/config.status
 	$(AM_V_GEN)sed                                          \
