diff --git a/config/init/systemd/Makefile.am b/config/init/systemd/Makefile.am
index 6e2bb13..ee4af79 100644
--- a/config/init/systemd/Makefile.am
+++ b/config/init/systemd/Makefile.am
@@ -3,11 +3,7 @@ EXTRA_DIST = \
 	lxc.service
 
 if INIT_SCRIPT_SYSTEMD
-if HAVE_DEBIAN
 SYSTEMD_UNIT_DIR = /lib/systemd/system
-else
-SYSTEMD_UNIT_DIR = /usr/lib/systemd/system
-endif
 
 install-systemd: lxc.service lxc-devsetup
 	$(MKDIR_P) $(DESTDIR)$(SYSTEMD_UNIT_DIR)
