diff -Nur lxc-0.9.0.old/lxc-templates-3.0.3/templates/lxc-openmandriva.in lxc-0.9.0/lxc-templates-3.0.3/templates/lxc-openmandriva.in
--- lxc-0.9.0.old/lxc-templates-3.0.3/templates/lxc-openmandriva.in	2013-12-24 14:13:59.000000000 +0400
+++ lxc-0.9.0/lxc-templates-3.0.3/templates/lxc-openmandriva.in	2013-12-24 20:22:59.755018844 +0400
@@ -118,9 +118,12 @@
     # download a mini openmandriva into a cache
     echo "Downloading openmandriva minimal ..."
     URPMI="/usr/sbin/urpmi.addmedia --urpmi-root $INSTALL_ROOT main http://abf.rosalinux.ru/downloads/$release/repository/$arch/main/release"
+    URPMI2="/usr/sbin/urpmi.addmedia --urpmi-root $INSTALL_ROOT main_updates http://abf.rosalinux.ru/downloads/$release/repository/$arch/main/updates"
     echo $URPMI
+    echo $URPMI2
     URPMI_BASE="/usr/sbin/urpmi --no-suggests --no-verify-rpm --ignorearch --root $INSTALL_ROOT --urpmi-root $INSTALL_ROOT --auto $PKG_LIST"
     $URPMI
+    $URPMI2
     $URPMI_BASE
     # We're splitting the old loop into two loops plus a directory retrival.
     # First loop...  Try and retrive a mirror list with retries and a slight
